Fix #873

zenhack opened the pull request.

naved001 commented:
* it looks like the linter is replacing `:%VIRTUAL_ENV%/` in `ci/wsgi.conf` with a real value.

* also, we should only run this test [`if DB == sqlite`](https://github.com/CCI-MOC/hil/blob/master/ci/pep8_tests.sh#L4)

zenhack commented:
It was actually the existing apache CI stuff that was modifying wsgi.conf; I changed that so it shouldn't modify the one in the source tree anymore. Fixed the SQLite thing too.
